src/test/java/org/gridsuite/study/server/rootnetworks/RootNetworkTest.java

```diff
@@ -551,35 +544,68 @@ void testDeleteRootNetworkFailed() throws Exception {
         assertEquals(2, studyService.getExistingBasicRootNetworkInfos(studyEntity.getId()).size());
     }
 
+    @Test
+    void testUpdateRootNetworkNameAndTag() throws Exception {
```

Comment:
Maybe add 2 tests where you change only name or only tag.
It doesn't cost a lot and it will avoid future regressions